actions/platform/integrations/add_integration.py

```diff
@@ -88,12 +97,16 @@ async def run(
             if len(resolved_match) > 0:
                 for resolved_slot in resolved_match:
                     events.append(SlotSet(resolved_slot, resolved_match[resolved_slot]))
+                if resolved_match["integration_setup_kind"] == "cloud":
```

Comment:
I think having these flow_finished_count in random spots is strange, maybe we should change this over to reset the slots on starting the flow, and have a action at the end send metrics based on them

Reply:
I agree, it's strange. Something else would be to have a function that processes the happy path and just call that function. 

I could be wrong, but adding more actions would require an additional slot to store the outcome of the API call, plus adding an additional step and so on. Lets see how this behaves.

Reply:
Yeah you're right. Shipping it :)